doc/plugin_server_keymanager_aws_kms.md

```diff
@@ -6,13 +6,15 @@ The `aws_kms` key manager plugin leverages the AWS Key Management Service (KMS)
-|-------------------|--------|---------------------------------------|-------------------------------------------------------------------------------|---------------------------------------------------------|
-| access_key_id     | string | see [AWS KMS Access](#aws-kms-access) | The Access Key Id used to authenticate to KMS                                 | Value of the AWS_ACCESS_KEY_ID environment variable     |
-| secret_access_key | string | see [AWS KMS Access](#aws-kms-access) | The Secret Access Key used to authenticate to KMS                             | Value of the AWS_SECRET_ACCESS_KEY environment variable |
-| region            | string | yes                                   | The region where the keys will be stored                                      |                                                         |
-| key_metadata_file | string | yes                                   | A file path location where information about generated keys will be persisted |                                                         |
-| key_policy_file   | string | no                                    | A file path location to a custom key policy in JSON format                    | ""                                                      |
+| Key                  | Type   | Required                              | Description                                                                                | Default                                                 |
+|----------------------|--------|---------------------------------------|--------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| access_key_id        | string | see [AWS KMS Access](#aws-kms-access) | The Access Key Id used to authenticate to KMS                                              | Value of the AWS_ACCESS_KEY_ID environment variable     |
+| secret_access_key    | string | see [AWS KMS Access](#aws-kms-access) | The Secret Access Key used to authenticate to KMS                                          | Value of the AWS_SECRET_ACCESS_KEY environment variable |
+| region               | string | yes                                   | The region where the keys will be stored                                                   |                                                         |
+| key_metadata_file    | string | Required if key_identifier_value is not set | A file path location where information about generated keys will be persisted (deprecated, use key_identifier_file instead) |                                                         |
+| key_identifier_file  | string | Required if key_identifier_value is not set | A file path location where information about generated keys will be persisted              |                                                         |
```

Comment:
Are both of these files required when `key_identifier_value` is not set?

Reply:
Apologies, I missed the deprecated notice. You may consider removing more completely e.g.

```suggestion
| key_metadata_file    | string | Deprecated. Use key_identifier_file instead |                                                         |
| key_identifier_file  | string | Required if key_identifier_value is not set | A file path location where information about generated keys will be persisted              |                                                         |
```

Or

```suggestion
| key_identifier_file  | string | Required if key_identifier_value is not set | A file path location where information about generated keys will be persisted (previously `key_metadata_file`)             |                                                         |
```

Reply:
`key_metadata_file` is deprecated and replaced with `key_identifier_file`. So maybe the required column for `key_metadata_file` should just say "no"?

Reply:
>So maybe the required column for key_metadata_file should just say "no"?

Sounds reasonable to me.

Reply:
Done.